platforms/core-configuration/model-core/src/main/java/org/gradle/api/internal/model/NamedObjectInstantiator.java

```diff
@@ -282,6 +280,10 @@ private void visitFields(Class<?> type, ValidationProblemCollector collector) {
         }
     }
 
+    private LoadingCache<String, Object> cashFactory(Class<?> type) {
```

Comment:
🤔 You have a `cash` factory?

Reply:
wait for it :).
just kidding, I'll fix this :)